.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,37 @@
+name: Continuous Integration
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '42 15 * * *'
+jobs:
+  build:
+    name: HHVM ${{matrix.hhvm}} - ${{matrix.os}}
+    strategy:
+      # Run tests on all OS's and HHVM versions, even if one fails
+      fail-fast: false
+      matrix:
+        os: [ ubuntu ]
+        hhvm:
+          - '4.80'
+          - latest
+          - nightly
+    runs-on: ${{matrix.os}}-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: hhvm/actions/hack-lint-test@master
+        with:
+          hhvm: ${{matrix.hhvm}}
+      # Check the codegen hasn't been modified; this is also done as part of the
+      # next step, but we want to make sure the verify-signatures executable works
```

Comment:
we should probably end with an intentional + inverted failure :)

Reply:
> this is a dev tool

Which of our projects is not? :D

> we should probably end with an intentional + inverted failure :)

What do you mean, should I swap the order of those 2 steps?

Reply:
Add a new step:

```
echo >> some/codegen/file
bin/verify-codegen
```

... but expect failure then

Reply:
https://github.com/jjergus/hack-codegen/runs/1751696414